client/js/components/comments/comment-list.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import moment from 'moment';
+
+class CommentListItem extends React.Component{
+  render() {
+    return (
+      <li style={{position: 'relative', padding: '10px 0'}} key={this.props.comment.id}>
+        <span>{this.props.comment.content}</span>
```

Comment:
@vrael560 It is ok, but didn't we agree that comments will accept markdown?
You can check how to display content of comment as markdown in snippet componet.